src/LibraryManager.Vsix/Json/Completion/CompletionController.cs

```diff
@@ -173,7 +173,7 @@ private JSONParseItem GetItemBeforePosition(int pos, JSONEditorDocument document
                     if (item is JSONComplexItem)
                     {
```

Comment:
use `if (item is JSONComplexItem complexItem)` here so you don't have to cast below.

Reply:
Will do